src/docker-images/azure-blob-adapter/wsgi.py

```diff
@@ -24,37 +24,51 @@ def application(request):
     :param PlainRequest request:
     '''
     try:
-        blob_name = request.headers['X-Tag']
-    except KeyError:
-        logger.exception('Key Error')
-        return Response(status=400)
+        if request.method == 'GET' and request.path == '/healthz':
+            def get_container():
+                container = append_blob_service.get_container_properties(
+                    container_name=container_name)
+                logger.info('Successfully get container {}: {}'.format(
+                    container_name, container.properties.etag))
 
-    def append_blob():
-        resource_properties = append_blob_service.append_blob_from_bytes(
-            container_name=container_name,
-            blob_name=blob_name,
-            blob=request.get_data(),
-            count=request.content_length)
-        logger.info('Successfully append {} bytes to blob {} in container {}: {}'.format(
-            request.content_length, blob_name, container_name, resource_properties.etag))
-
-    def create_blob():
-        resource_properties = append_blob_service.create_blob(
-            container_name=container_name,
-            blob_name=blob_name)
-        logger.info('Successfully create blob {} in container {}: {}'.format(
-            blob_name, container_name, resource_properties.etag))
+            get_container()
+            return Response(status=200)
 
-    try:
-        try:
-            append_blob()
-        except AzureMissingResourceHttpError:
-            create_blob()
-            append_blob()
-        return Response(status=201)
-    except AzureHttpError:
-        logger.exception('Azure HTTP Error')
-        return Response(status=502)
+        if request.method == 'POST' and request.path == '/':
+            try:
+                blob_name = request.headers['X-Tag']
+            except KeyError:
+                logger.exception('Key Error')
+                return Response(status=400)
+
+            def append_blob():
+                resource_properties = append_blob_service.append_blob_from_bytes(
+                    container_name=container_name,
+                    blob_name=blob_name,
+                    blob=request.get_data(),
+                    count=request.content_length)
+                logger.info('Successfully append {} bytes to blob {} in container {}: {}'.format(
```

Comment:
Do we really need such log in production?

Reply:
You're right, since I enabled azure internal info logs, all of these info are directly outputted by azure library